src/main/kotlin/org/wycliffeassociates/otter/common/domain/ImportResourceContainer.kt

```diff
@@ -143,19 +127,33 @@ class ImportResourceContainer(
     }
 
     private fun getCategories(rc: ResourceContainer): List<Collection> {
-        return arrayListOf(
-                Collection(1, "bible-ot", "bible-ot", "Old Testament", null),
-                Collection(2, "bible-nt", "bible-nt", "New Testament", null)
-        )
+        val collections = arrayListOf<Collection>()
+        val config = rc.config
+        config.let {
```

Comment:
Do we need `config.let` here?

Reply:
totally forgot the ?, and as such, also forgot it needs to change to it

config is nullable in the resource container

Reply:
Not sure, but since you are doing the type check, you might be able to drop the let entirely. You've already got a local `config` variable, so kotlin should be able to do a smart cast from `Config?` to `OtterResourceContainerConfig`?
```kotlin
if (config is OtterResourceContainerConfig) {
    config.extendedDublinCore?.let {
        for (cat in it.categories) {
            collections.add(
                    Collection(cat.sort, cat.identifier, cat.identifier, cat.title, null)
            )
        }
    }
}
```